build/Targets/VSL.Imports.targets

```diff
@@ -392,8 +392,11 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
 
-    <Message Text="Adding optimization data to @(IntermediateAssembly)"/>
-    <Exec Command="&quot;$(IbcMergePath)&quot; -q -f -partialNGEN -minify -mo &quot;@(IntermediateAssembly)&quot; -incremental &quot;$(OptimizationDataFile)&quot;" />
+    
+    <!-- SKIPPING: https://github.com/dotnet/roslyn/issues/12508
+      <Message Text="Adding optimization data to @(IntermediateAssembly)"/>
+      <Exec Command="&quot;$(IbcMergePath)&quot; -q -f -partialNGEN -minify -mo &quot;@(IntermediateAssembly)&quot; -incremental &quot;$(OptimizationDataFile)&quot;" />
+    -->
```

Comment:
I suspect we won't be able to insert until this is dealt with, but :+1: to unblock PRs.